tests/unit/cmd/sca/test_ci.py

```diff
@@ -0,0 +1,45 @@
+from unittest.mock import Mock, patch
+
+import click
+
+from ggshield.cmd.main import cli
+from ggshield.sca.sca_scan_models import SCAScanDiffOutput
+
+
+@patch("ggshield.cmd.sca.scan.ci.sca_scan_diff")
+@patch("ggshield.cmd.sca.scan.ci.get_commits")
+@patch("ggshield.cmd.utils.check_git_dir")
+def test_ci_cmd(
```

Comment:
You could rename it `test_sca_scan_ci_no_commit` as it basically test that everything work if there are no commit to scan